mesa/__init__.py

```diff
@@ -26,5 +26,5 @@
 __title__ = "mesa"
 __version__ = "3.2.0.dev"
 __license__ = "Apache 2.0"
-_this_year = datetime.datetime.now(tz=datetime.UTC).date().year
+_this_year = datetime.datetime.now(tz=datetime.timezone.utc).date().year
```

Comment:
Mesa currently supports Python 3.11 and above. That's enforced by pre-commit, which is why it's changed back every time.

See https://github.com/projectmesa/mesa/blob/5166670b33895f639b60e895a95e1591b6f0f0d7/pyproject.toml#L28